packages-exp/functions-exp/src/index.node.ts

```diff
@@ -16,10 +16,22 @@
  */
 import { registerVersion } from '@firebase/app-exp';
 import { registerFunctions } from './config';
-import 'isomorphic-fetch';
+import 'whatwg-fetch';
```

Comment:
Instead of including a new polyfill, we can replace `Header` with a string map. I prefer to not include a new polyfill that only provides `Header` and given that the workaround is really simple.

Reply:
Fixed (it is a string-keyed object, `fetch` signature does not accept Maps).